src/pages/playerUrls.js

```diff
@@ -874,4 +874,10 @@ module.exports = {
   vixcloud: {
     match: ['*://vixcloud.cc/*', '*://vixcloud.co/*'],
   },
+  // otakuanimes
+  otakuanimes: {
+    match: [
+      '*://otakuanimess.com/episodio/*', 
+      '*://ns*'],
```

Comment:
This is not a valid domain

Reply:
Which domain is not valid? The `ns*`?

This domain has several variations because it is generated automatically by the platform, so it is impossible without a regex.
It's impossible to map everything.

If you can, explain to me what I should actually put here.

Reply:
https://developer.chrome.com/docs/extensions/mv2/match_patterns/

Chrome does not allow such patterns

Reply:
I changed the pattern and test the glob pattern in this site:

https://www.digitalocean.com/community/tools/glob?comments=true&glob=%2A%3A%2F%2Fns%2A%2F%2A%2F%2A%2F%2A%2F%2A&matches=false&tests=https%3A%2F%2Fns558972.ip-54-39-50.net%2FAnimes%2FK%2Fkonyaku-haki-sareta-reijou-wo-hirotta-ore-ga-ikenai-koto-wo-oshiekomu%2F04.mp4